src/System.Windows.Forms/src/System/Windows/Forms/MonthCalendar.MonthCalendarAccessibleObject.cs

```diff
@@ -26,9 +25,10 @@ internal class MonthCalendarAccessibleObject : ControlAccessibleObject
 
         public MonthCalendarAccessibleObject(MonthCalendar owner) : base(owner)
         {
-            _owningMonthCalendar = owner;
-
-            _owningMonthCalendar.DisplayRangeChanged += OnMonthCalendarStateChanged;
+            if(this.TryGetOwnerAs(out MonthCalendar? owningMonthCalendar))
+            {
+                owningMonthCalendar.DisplayRangeChanged += OnMonthCalendarStateChanged;
+            }
```

Comment:
Can we do it this way?
```suggestion
            owner.DisplayRangeChanged += OnMonthCalendarStateChanged;
```